packages/theme/src/cli/services/update/init.ts

```diff
@@ -0,0 +1,47 @@
+import {schemaUrlV1} from './schemas/update_extension_schema_v1.js'
+import {AbortError} from '@shopify/cli-kit/node/error'
+import {fileExists, writeFile} from '@shopify/cli-kit/node/fs'
+import {joinPath} from '@shopify/cli-kit/node/path'
+import {renderSuccess} from '@shopify/cli-kit/node/ui'
+
+export async function init(rawPath?: string) {
+  const path = joinPath(rawPath ?? '.', 'update_extension.json')
+
+  await createScript(path)
+
+  renderSuccess({
+    body: [`The '${path}' script has been created.`],
```

Comment:
/nit is this the full path or relative path? relative path may be a better experience